PyDSS/pydss_project.py

```diff
@@ -270,7 +270,7 @@ def run(self, logging_configured=True, tar_project=False, zip_project=False):
         self._simulation_config["Logging"]["Pre-configured logging"] = logging_configured
         store_filename = os.path.join(self._project_dir, STORE_FILENAME)
         complevel = 0
-        if self._simulation_config["Exports"]["Export Compression"]:
+        if self._simulation_config["Exports"].get("Export Compression", False):
             complevel = 9
```

Comment:
Can you please add comments on why compression levels are hardcoded to either 0 or 9

Reply:
Sure, that's fair.  I'll add a comment.  The HDFStore options are below.  Do you want to provide any additional granularity to the user?

```
complevel : int, 0-9, default None
        Specifies a compression level for data.
        A value of 0 or None disables compression.
complib : {'zlib', 'lzo', 'bzip2', 'blosc'}, default 'zlib'
        Specifies the compression library to be used.
        As of v0.20.2 these additional compressors for Blosc are supported
        (default if no compressor specified: 'blosc:blosclz'):
        {'blosc:blosclz', 'blosc:lz4', 'blosc:lz4hc', 'blosc:snappy',
         'blosc:zlib', 'blosc:zstd'}.
        Specifying a compression library which is not available issues
        a ValueError.
fletcher32 : bool, default False
        If applying compression use the fletcher32 checksum
```

Reply:
This seems to be sufficient